apps/dav/lib/Controller/CalendarExportController.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+namespace OCA\DAV\Controller;
+
+use OCA\DAV\AppInfo\Application;
+use OCA\DAV\CalDAV\Export\ExportService;
+use OCP\AppFramework\Http;
+use OCP\AppFramework\Http\Attribute\ApiRoute;
+use OCP\AppFramework\Http\Attribute\NoAdminRequired;
+use OCP\AppFramework\Http\Attribute\OpenAPI;
+use OCP\AppFramework\Http\Attribute\UserRateLimit;
+use OCP\AppFramework\Http\DataResponse;
+use OCP\AppFramework\Http\StreamGeneratorResponse;
+use OCP\AppFramework\OCSController;
+use OCP\Calendar\CalendarExportOptions;
+use OCP\Calendar\ICalendarExport;
+use OCP\Calendar\IManager;
+use OCP\IGroupManager;
+use OCP\IRequest;
+use OCP\IUserManager;
+use OCP\IUserSession;
+
+class CalendarExportController extends OCSController {
+
+	public function __construct(
+		IRequest $request,
+		private IUserSession $userSession,
+		private IUserManager $userManager,
+		private IGroupManager $groupManager,
+		private IManager $calendarManager,
+		private ExportService $exportService,
+	) {
+		parent::__construct(Application::APP_ID, $request);
+	}
+	
+	/**
+	 * Export calendar data
+	 *
+	 * @param string $id calendar id
+	 * @param string|null $format data format
+	 * @param array{rangeStart:string,rangeCount:int<1,max>} $options configuration options
+	 * @param string|null $user system user id
+	 *
+	 * @return DataResponse<Http::STATUS_BAD_REQUEST|Http::STATUS_UNAUTHORIZED,array{error?:non-empty-string},array{}>|StreamGeneratorResponse<Http::STATUS_OK,array{Content-Type:string}>
```

Comment:
```suggestion
	 * @return DataResponse<Http::STATUS_BAD_REQUEST|Http::STATUS_UNAUTHORIZED, array{error?: non-empty-string}, array{}>|StreamGeneratorResponse<Http::STATUS_OK, array{Content-Type: string}>
```
You should try to add a few spaces to make it a bit more readable.